packages/upgrade/src/Rector/SimpleMethodChangesRector.php

```diff
@@ -47,6 +64,21 @@ public function getChanges(): array
                     'getSubNavigationPosition' => function (ClassMethod $node): void {
                         $node->flags &= Modifiers::STATIC;
                     },
+                    'getRoutePath' => $prependPanelParamModifier,
+                    'getRelativeRouteName' => $prependPanelParamModifier,
+                    'getSlug' => $prependPanelParamModifier,
```

Comment:
I think this one needs to prepend a nullable panel object param

Reply:
Sharp, just done!